### ISSUE TITLE
Runtime error on create new show

### ISSUE THREAD

**Issue by jordanhenshaw** [open-stage/blender-dmx#223]
### Please confirm that;

- [X] I have checked the [documentation](https://blenderdmx.eu/docs/faq/), and I didn't find a solution to my problem / an answer to my question.
- [X] I have searched the [issues](https://github.com/open-stage/blender-dmx/issues), and I didn't find a solution to my problem / an answer to my question.
- [X] I have asked in the user group on [Discord](https://discord.gg/FQVVyc45T9) for ideas and solutions and have waited for someone to respond.

### What is your Blender version?

4.1

### What operating system are you using Blender on?

macOS

### What happened?

Line 461 of dmx.py needs to ensure the Background node exists before accessing it. I happened to not have the background node in my World, I ran this addon for the first time and the first button I pressed was an immediate runtime error for "Background" node not existing.

This:

`scene.world.node_tree.nodes['Background'].inputs[0].default_value = (0,0,0,0)`

Should include to a try/except or a hasattr() or something of the like to learn whether the node needs to be added prior to accessing it. Otherwise you get runtime error if user happened to delete it for whatever reason.

**Comment by vanous** [comment]
Thank you for the report, appreciated. What would be the reason to have the background removed? Fixed [here](https://github.com/open-stage/blender-dmx/pull/224). Btw. i did not see this mentioned in the Discord even when you checked the form that you asked there.

**Comment by jordanhenshaw** [comment]
The form forced me to check that box, so I checked it. Wasn't really a question, was more of a report, since I have the exact same type of bugs in my stuff every day.

Background was removed because I use World for Alva Sorcerer, which uses nodes for lighting group controllers, not visualization compositing.

**Comment by jordanhenshaw** [comment]
Also, nice job breaking the rules on the programmer.py UI draw to get selective disabling, using 6-7 columns for what's a single row! I never did think of that.

**Comment by vanous** [comment]
> Background was removed because I use World for Alva Sorcerer, which uses nodes for lighting group controllers, not visualization compositing.

My solution was to actually re-create the nodes and add the Background, will this break your stuff? I thought of just checking/try-excepting, but then i decided to rather add the missing nodes. Please let me know.

> Also, nice job breaking the rules on the programmer.py UI draw to get selective disabling, using 6-7 columns for what's a single row! I never did think of that.

:) Which part of the programmer do you mean, the selection icons?

![image](https://github.com/user-attachments/assets/f145d470-faed-47c1-b176-329fc003ad70)

Is there some Blender recommendations on UI layout which i broke? :)

**Comment by jordanhenshaw** [comment]
Yes, I meant those icons, how it's able to disable individual buttons as opposed to the whole row. "Breaking rules" meaning stretching the limits of what bpy can do, in a very good way :)

No, my stuff doesn't conflict with yours, as long as you're always checking that things exist prior to trying to access.

I tried the new code and it works!
